agile_problem1_test.exs

```diff
@@ -0,0 +1,21 @@
+ExUnit.start
+
+defmodule Problem1Test do
+	use ExUnit.Case
+	
+	test "Problem 1: positive test 1" do

```

Comment:
Unit tests should have a more explicit naming, like a good writeup about this is [here](http://osherove.com/blog/2005/4/3/naming-standards-for-unit-tests.html), boils down to [UnitOfWork__StateUnderTest__ExpectedBehavior]